src/main/java/application/controllers/WindowFactory.java

```diff
@@ -69,9 +69,16 @@ public static FileChooser createGraphChooser() {
 
         File selectedFile = directoryChooser.showOpenDialog(window);
 
-        mainController.getGraphController().getGraph().getNodeMapFromFile(selectedFile.toString());
+        try {
+            mainController.getGraphController().getGraph().getNodeMapFromFile(selectedFile.toString());
+            mainController.initGraph();
+        }
+        catch (Exception e) {
+            System.out.println("You probably closed te windows before selecting.");
```

Comment:
printing an error message in the console is not neccessary as closing the window before selecting a file is a benign action